kubernetes/auth_test.go

```diff
@@ -0,0 +1,69 @@
+package kubernetes
+
+import (
+	"os"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func newFakeClient() *Config {
+	k := Config{}
+	k.clientset = fake.NewSimpleClientset()
+	return &k
+}
+
+func TestAuthenticationDefaultError(t *testing.T) {
+	k := newFakeClient()
+	err := k.Authenticate()
+	if err == nil {
+		t.Error(err)
+	}
```

Comment:
Consider using testify, e.g. https://github.com/TRON-US/go-common/blob/master/operator/string_test.go#L27 it makes checks cleaner (forces you to write a reason on each failure), and also consistent with the rest of the tests.

Reply:
Done!

Reply:
Done!